app.rb

```diff
@@ -0,0 +1,111 @@
+require_relative 'book'
```

Comment:
- [ ] Good job on implementing all requested methods in `app.rb`, but, I wasn't able to test it since the `main.rb` doesn't exist. Kindly add the `main.rb` as the entry point for your application as per this [requirement](https://github.com/microverseinc/curriculum-ruby/blob/main/oop/school_library_ui.md#:~:text=In%20your%20main.rb%20define%20the%20entry%20point%2C%20this%20will%20be%20a%20method%20called%20main%20that%20is%20invoked%20at%20the%20end%20of%20your%20file.%20This%20method%20should%20do%20the%20following%3A).